resources/docs/Documentation.md

```diff
@@ -115,7 +115,7 @@ RSS Guard is modular application which supports plugins. It offers well-maintain
 * [Nextcloud News](https://apps.nextcloud.com/apps/news) plugin: Nextcloud News is a Nextcloud app which adds feed reader abilities into your Nextcloud instances. Nextcloud is nearly perfect self-hosted artifact synchronization platform.
 * [Gmail](https://www.google.com/gmail) plugin: Yes, you are reading it right. RSS Guard can be used as very lightweight and simple e-mail client. This plugins uses [Gmail API](https://developers.google.com/gmail/api) and offers even e-mail sending.
 
-All plugins share almost all core RSS Guard's features, including labels, recycle bins, podcasts fetching or newspaper view. They are implemented in a very transparent way, making it easy to maintain them or add new ones.
+All plugins share almost all core RSS Guard's features, including labels, recycle bins, podcasts fetching or newspaper view. They are implemented in a very transparent way, making it easy to maintain them or add new ones. For more information on how to configure them please refer to the [Accounts](accounts.md) help document.
```

Comment:
Is there a specific reason to keep it all in a single document? I'm looking to expand on the database backends along with more information on what exactly the settings do as. I'm also looking at the `config.ini` file. I believe having a more thorough review of its settings be included in the main documentation file would be counter productive due to the potential length of it. A bigger file is also harder to maintain and hold in your mind as you work through it.

Reply:
Yes, there is specific reason, simplicity. I even considered to embed images into the documentation file via Base64/html technique but abandoned for it being inpractical.

One file is just best I think, it does not really harm anything, cross-link navigation works via anchors etc.

As for config.ini, yes, config.ini is important, you can see what exact keys the settings use and what default values are rougly in: https://github.com/martinrotter/rssguard/blob/master/src/librssguard/miscellaneous/settings.cpp#L19

If you feel that it would be long or over-technical, just include it in "Misc" section which is there just for these reasons and again, I believe we do not need to document every single possible setting as many of them are self-explanatory, just describe common concepts and caveats.